django/contrib/admin/static/admin/js/nav_sidebar.js

```diff
@@ -15,14 +15,15 @@
         }
 
         const main = document.getElementById('main');
+        const navSidebar = document.getElementById('nav-sidebar');
         let navSidebarIsOpen = localStorage.getItem('django.admin.navSidebarIsOpen');
         if (navSidebarIsOpen === null) {
             navSidebarIsOpen = 'true';
         }
         if (navSidebarIsOpen === 'false') {
             disableNavLinkTabbing();
         }
-        main.classList.toggle('shifted', navSidebarIsOpen === 'true');
+        main.classList.toggle('shifted', navSidebar && navSidebarIsOpen === 'true');
```

Comment:
`nav_sidebar.js` is loaded only when admin's sidebar is enabled so these changes are unnecessary. Moreover we reach this code only when `toggle-nav-sidebar` exists.